doc/redirects.json

```diff
@@ -3223,15 +3223,24 @@
   "manifest-file-via-maven-plugin": [
     "index.html#manifest-file-via-maven-plugin"
   ],
-  "nim": [
```

Comment:
This File is for backwards stability of urls. Since you changed the heading id people having the old ref should be redirected. But you removed the id entirely. 
This breaks their old links.

Reply:
Well some CI complained and instructed me to put links into that file so I did. Links aren't totally broken, just the fragment.